### ISSUE TITLE
not downloading

### ISSUE THREAD

**Issue by itsonlym3** [jarulsamy/Twitter-Archive#1]
i can't seem to get this to work. any ideas as to why? when i run the script, i get like to images and that's it.

**Comment by nickk** [comment]
you're only getting images? isn't that what the readme says?

**Comment by jarulsamy** [comment]
> i can't seem to get this to work. any ideas as to why? when i run the script, i get like to images and that's it.

Apologies for getting to this so late. This current version was only ever intended to download images by scraping the source of the each tweet using `selenium`. Twitter obfuscates most of their HTML, so it is extremely likely that this script is broken right now. I currently don't have the time to rewrite it. I'm waiting until the official API route is introduced. That way, I can hopefully write a better all around solution that saves more than just images. 

The latest update on the [Twitter feedback forum](https://twitterdevfeedback.uservoice.com/forums/930250-twitter-api/suggestions/39678766-api-endpoint-for-getting-bookmarks) from Andy Piper on Aug 10, 2021:
>This is being worked on. There are some dependencies that are higher on the backlog that need to be completed first.

Hopefully, in a few months, once this new API is introduced, I will be able to update this.

**Comment by FarisHijazi** [comment]
Hello, I'm working on a similar solution but it's usually easier in javascript running it within the browser

also I've noticed that your code won't scroll down and thus won't see all the outputs


The one I have here takes care of all these issues, still working on making it download everything other than images too

https://github.com/FarisHijazi/TwitterDownloader

**Comment by jarulsamy** [comment]
The new Twitter API has finally been introduced. I have updated this application to use the new Bookmark APIs, and for the time being everything seems to be working correctly. Things are still in the very early stages, but so far I have media downloading without any hitches. I plan to update the README with new instructions in the next few days. 

I will go ahead and close this issue, since it is no longer relevant given that the entire app has been rewritten.

Thanks again for your interest in this project!
